src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,45 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { useTodos } from '../../context/TodoProvider';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
 
 export const TodoModal: React.FC = () => {
+  const {
+    activeTodo, setActiveTodo, setIsPressed, isLoading, setIsLoading,
+  } = useTodos();
+
+  const [userOfTodo, setUserOfTodo] = useState<User | null>(null);
+
+  useEffect(() => {
+    if (!activeTodo) {
+      return;
+    }
+
+    getUser(activeTodo.userId).then(data => {
+      setUserOfTodo(data);
+      setIsLoading(false);
+    });
+  }, [activeTodo]);
```

Comment:
React Hook useEffect has a missing dependency: 'setIsLoading'. Either include it or remove the dependency array